template/src/main/java/com/simtlix/techgroups/template/controllers/CustomerController.java

```diff
@@ -0,0 +1,84 @@
+package com.simtlix.techgroups.template.controllers;
+
+import com.simtlix.techgroups.template.model.Customer;
+import com.simtlix.techgroups.template.services.CustomerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+/**
+ * Created by Facundo on 1/29/2018.
+ */
+@RestController
+@RequestMapping("/customers")
+public class CustomerController {
+
+    private CustomerService customerService;
+
+    @Autowired
+    public CustomerController(CustomerService customerService){
+        this.customerService = customerService;
+    }
+
+    @RequestMapping(method = RequestMethod.GET, produces = "application/JSON")
+    public ResponseEntity<List<Customer>> getCustomers() {
+        ResponseEntity<List<Customer>> responseEntity;
+        List<Customer> customers = customerService.getCustomers();
+        if (customers == null || customers.isEmpty()) {
+           responseEntity = new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
Es mejor devolver una lista vacía. Un NOT FOUND puede confundir con que el web service no existe

Reply:
Agree. Se podría mandar un OK o un NO_CONTENT.